src/features/embalm/stepContent/steps/SelectArchaeologists.tsx

```diff
@@ -59,24 +61,61 @@ export function SelectArchaeologists() {
     setCurrentPage(1);
   };
 
+  function handlePageSizeChange(newValue: IPageSizeSetByOption | null) {
+    const newPageSize = newValue!.value;
+    setPageSize(newPageSize!);
+    setPaginationSize(newPageSize);
+  }
+
   return (
     <Flex
       direction="column"
       width="100%"
     >
-      <Heading>Select Archaeologists</Heading>
+      <Heading>Archaeologists</Heading>
       <Text
         variant="primary"
-        mt="6"
+        mt="4"
+        fontSize="16px"
       >
         Resurrection Time
       </Text>
-      <Text
-        variant="primary"
-        mt="2"
-      >
-        Currently set: 09.22.22 7:30pm (edit)
-      </Text>
+      <HStack>
+        <>
+          <Text
+            variant="primary"
+            color="brand.600"
+          >
+            Currently set:{' '}
+            <Text
+              as="u"
+              color="brand.600"
+            >
+              {moment(resurrectionDate).format('DD.MM.YY h:mma')}
+            </Text>
+          </Text>
+
+          <Text
+            as="i"
+            color="brand.900"
+            onClick={() => {
+              setResurrectionTimeEdit(prevValue => !prevValue);
+            }}
+            cursor="pointer"
+            _hover={{
+              textDecoration: 'underline',
+            }}
+          >
+            ({resurrectionTimeEdit ? 'hide' : 'edit'})
+          </Text>
+        </>
+      </HStack>
+      {resurrectionTimeEdit && (
+        <SetResurrection
+          mt={5}
+          w={'50%'}
```

Comment:
I think that the width here in this case would be better as a fixed width for now. At 50% the radio labels wrap at certain screen sizes. Maybe something like `w='500px'`.